### PR TITLE
Use a formula to closely match current numbers for lmp.

### PR DESCRIPTION
**Pull request by manyoso**

### CODE REVIEW

**Comment by manyoso** [comment]
ELO   | 4.82 +- 3.88 (95%)
SPRT  | 10.0+0.1s Threads=1 Hash=8MB
LLR   | 2.98 (-2.94, 2.94) [-1.00, 4.00]
Games | N: 11168 W: 2106 L: 1951 D: 7111

ELO   | 1.75 +- 1.73 (95%)
SPRT  | 60.0+0.6s Threads=1 Hash=64MB
LLR   | 3.04 (-2.94, 2.94) [-1.00, 4.00]
Games | N: 37648 W: 4704 L: 4514 D: 28430

File: src/search.c [hunk 1]
```diff
@@ -49,12 +49,19 @@ volatile int ABORT_SIGNAL; // Global ABORT flag for threads
...
 void initSearch() {
 
     // Init Late Move Reductions Table
     for (int depth = 1; depth < 64; depth++)
         for (int played = 1; played < 64; played++)
             LMRTable[depth][played] = 0.75 + log(depth) * log(played) / 2.25;
+
+    for (int depth = 0; depth < 9; ++depth) {
+        LateMovePruningCounts[0][depth] = roundf((depth > 0 ? 2.5 : 0) + depth * depth * 2 / 4.5);
+        LateMovePruningCounts[1][depth] = roundf((depth > 0 ?   4 : 0) + depth * depth * 4 / 4.5);
```
Comment (AndyGrant) [inline src/search.c]: * The first line has ``2.5 : 0``, and the next has ``4 : 0``. Make the second line ``4.0 : 0`` to match the precision. 
* The end of each line has a 2x and a 4x multiplayer. Can you move that to the front, so it reads ``2 * depth * depth / 4.5`` to match other cases in the code if I recall.

Comment (AndyGrant) [reply src/search.c]: Change ``++depth`` to ``depth++`` since I always use ``i++`` in most cases in Ethereal

Comment (AndyGrant) [reply src/search.c]: Lets clean up the stack of includes/globals to be:

```
int LMRTable[64][64];                    
int LateMovePruningCounts[2][9]; 

volatile int ABORT_SIGNAL; // Global ABORT flag for threads
volatile int IS_PONDERING; // Global PONDER flag for threads
volatile int ANALYSISMODE; // Whether to make some changes for Analysis
```

I dropped the comment from LMRTable. LMR is ubiquitous. LMP, FMP, CMP, SNMP, those are not. but in this case we have Late Move Pruning fully typed out.

**Review by AndyGrant — CHANGES_REQUESTED** [review]

**Pull request merged**